pystruct/learners/ssvm.py

```diff
@@ -17,6 +29,39 @@ def __init__(self, model, max_iter=100, C=1.0, verbose=0,
         self.show_loss_every = show_loss_every
         self.n_jobs = n_jobs
         self.logger = logger
+        self.use_threads = use_threads
+        self.use_memmapping_pool = use_memmapping_pool
+        self.memmapping_temp_folder = memmapping_temp_folder 
+        ## spawn pool in init, every instance should have pool attribute
+        self._spawn_pool()
```

Comment:
I actually have an idea on how to handle this better and still meet scikit-learn standards. I'll write it up quickly later today.